CMakeLists.txt

```diff
@@ -141,10 +141,22 @@ endif(${KRATOS_EXCLUDE_OPENMP} MATCHES ON)
 set(Boost_USE_STATIC_LIBS   OFF)
 set(Boost_USE_MULTITHREADED ON)
 set(Boost_REALPATH ON)
-find_package(Boost COMPONENTS python REQUIRED)
+
+# Select python3 by default unless its specified in the configure file
+message("Using component: python${BOOST_PYTHON_SUFIX}")
+if(BOOST_PYTHON_SUFIX)
+  FIND_PACKAGE(Boost COMPONENTS "python${BOOST_PYTHON_SUFIX}" REQUIRED)
+else(BOOST_PYTHON_SUFIX)
+  FIND_PACKAGE(Boost COMPONENTS python REQUIRED)
+endif(BOOST_PYTHON_SUFIX)
```

Comment:
All this if-else block can be collapsed to `FIND_PACKAGE(Boost COMPONENTS "python${BOOST_PYTHON_SUFIX}" REQUIRED)` but probably this way is clearer. What do you think?